### DISCUSSION TITLE
Search according to input and return data

### DISCUSSION THREAD

**Discussion by AdministratedAdmin** [community/community#76797] [Programming Help] [answered]
### Body

How can I fill the form with data brought from the database with an input parameter?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by lyfeyvutha** [comment, marked as answer]
If you are referring to SQL commands, I recommend utilizing SELECT * dual, its such a game changer for me in inserting data all at once. I don't recommend you inserting it manually.

**Comment by Join555dgh** [comment]
Прослушал телефона
